hls4ml/templates/vivado/build_prj.tcl

```diff
@@ -149,19 +149,19 @@ set CSIM_RESULTS "./tb_data/csim_results.log"
 set RTL_COSIM_RESULTS "./tb_data/rtl_cosim_results.log"
 
 if {$opt(reset)} {
-  open_project -reset ${myproject}_prj
+    open_project -reset ${myproject}_prj
 } else {
-  open_project ${myproject}_prj
+    open_project ${myproject}_prj
 }
-set_top myproject
-add_files firmware/myproject.cpp -cflags "-std=c++0x"
-add_files -tb myproject_test.cpp -cflags "-std=c++0x"
+set_top ${myproject}
```

Comment:
If we're using a variable now, why are we still calling it `myproject` and not something more descriptive, like `project_name`?